lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -365,34 +330,6 @@ private IdentityVerificationService createIdentityVerificationService(
                 new FormDataValidator(), serviceFactory.getEventProbe());
     }
 
-    private DocumentCheckVerificationResult executeFallbackRequest(
-            DrivingPermitForm drivingPermitFormData) throws Exception {
-        ThirdPartyAPIService fallbackThirdPartyService =
-                selectThirdPartyAPIService(false, false, drivingPermitFormData.getLicenceIssuer());
-        eventProbe.counterMetric(DL_FALL_BACK_EXECUTING);
-        return identityVerificationService.verifyIdentity(
-                drivingPermitFormData, fallbackThirdPartyService);
-    }
-
-    private DocumentCheckVerificationResult executeFallbackIfDocumentFailedToVerify(
-            DocumentCheckVerificationResult documentDataVerificationResult,
-            DrivingPermitForm drivingPermitFormData)
-            throws Exception {
-        if (!documentDataVerificationResult.isVerified()
-                || (documentDataVerificationResult.getContraIndicators() != null
-                        && !documentDataVerificationResult.getContraIndicators().isEmpty())) {
-            LOGGER.info(
-                    "Document has been marked unverified during fallback window. Executing request with Direct connection");
-            documentDataVerificationResult = executeFallbackRequest(drivingPermitFormData);
-            if (documentDataVerificationResult.isVerified()) {
-                eventProbe.counterMetric(DL_VERIFICATION_FALLBACK_DEVIATION);
-                LOGGER.warn(
-                        "Document has been verified using DCS that failed verification using Direct connection");
-            }
-        }
-        return documentDataVerificationResult;
-    }
-
     private ThirdPartyAPIService selectThirdPartyAPIService(
             boolean dvaDirectEnabled, boolean dvlaDirectEnabled, String licenseIssuer) {
 
```

Comment:
Will we remove the else in the selctThirdPartyAPIService that defaults journeys to dcs?

Reply:
Yeah thats a fair call ill fix that now